templates/archives/stories/women_in_symbols.jinja2

```diff
@@ -0,0 +1,39 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+Women in Symbols
+{% endblock %}
+
+{% block story_body %}
+<p>Until recently, women have gone unnamed in many of their historical contributions, which is visible in this archive.
+    Most of the historical materials we can reference here to learn about early computing have been signed by men, with little evidence of the women behind them.
+    For example, a majority of the letters in this archive have been written by a secretary, whose initials only appear at the end of the letter, such as in the picture to the right, where FMV has signed the letter and NB has written it up.</p>
+
+<figure class="figure float-right">
+    <img src="{{ static('img/stories/women_in_symbols.png') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="A letter subscript above the cc line detailing those involved in its writing" />
+</figure>
+
+<p>However, internal documents sometimes display a portion of the work women performed in computing.
+    In the case of the choices made for which symbols computers would use, this archive demonstrates that women were heavily involved.
+    In a memo submitted to Laura by Edna Verzuh, there is a page of symbols which appear to await her commentary.
```

Comment:
If the women are only named by their first names, I would include a sentence that that is the case. It seems jarring to have the men's full names in this write up but only the first names of the women.